static-site/components/splash/index.tsx

```diff
@@ -0,0 +1,457 @@
+          {"Nextstrain".split("").map((letter, i) => (
+            <span style={{ color: `var(--titleColor${i})` }} key={i}>
+              {letter}
+            </span>
+          ))}
+        </div>
+      </div>
+
+      <div className={styles.headingSubtitle}>
+        Real-time tracking of pathogen evolution
+      </div>
+    </div>
+  );
+}
+
+/** A helper component to display the "narrative" icon on a group tile */
+function NarrativeIcon(): React.ReactElement {
+  return (
+    <TooltipWrapper description="Analysis in the form of a narrative">
+      <img
+        className={styles.infoIconImg}
+        src={bookOpenIcon.src}
+        alt="narrative"
+      />
+    </TooltipWrapper>
+  );
+}
+
+/**
+ * A helper React Component to render a section entry below the title
+ * of the home page, and above the expandable tiles section
+ */
+function Section({
+  abstract,
+  id,
+  imgSrc,
+  link,
+  title,
+}: {
+  /** the text to display under the section title */
+  abstract: string;
+  /** the unique id of the section */
+  id: string;
+  /** an icon image to adorn the section title with */
+  imgSrc: string;
+  /** the URL the section title should link to */
+  link: string;
+  /** title of the section */
+  title: string;
+}): React.ReactElement {
+  return (
+    <div
+      id={id}
+      className="col-12 col-md-6 col-lg-4"
+      style={{ paddingBottom: "20px" }}
+    >
+      <div style={{ display: "flex", flexDirection: "column", height: "100%" }}>
+        <a href={link}>
+          <img className={styles.sectionImage} src={imgSrc} alt={title} />
+          <span className={styles.sectionHeader}>{title}</span>
+        </a>
+
+        <div className={styles.sectionAbstract} style={{ flexGrow: 1 }}>
+          {abstract}
+        </div>
+
+        <BigSpacer />
+      </div>
+    </div>
+  );
+}
+
+/** A helper React Component to render one tile in the <ExpandableTile> component */
+function Tile({
+  tile,
+}: {
+  /** the Tile to render */
+  tile: SplashTile;
+}): React.ReactElement {
+  /** Narrative detection works for all three sources:
+   * 1. Core: /narratives/<narrative path>
+   * 2. Community: /community/narratives/<owner>/<narrative path>
+   * 3. Groups: /groups/<group>/narratives/<narrative path>
+   *
+   * Including slashes in the check prevents false positives.
+   */
+  const isNarrative: boolean = tile.url.includes("/narratives/");
+
+  let tileImg: string;
+  try {
+    tileImg = require(`../../static/pathogen_images/${tile.img}`).default.src;
+  } catch {
+    tileImg = emptyPathogenImage.src;
+  }
+
+  return (
+    <div
+      className={styles.tileOuter}
+      style={{ height: `${tileHeight}px`, width: `${tileWidth}px` }}
+    >
+      <div className={styles.tileInner}>
+        <a href={tile.url}>
+          <div className={styles.tileName}>{tile.name}</div>
+
+          <div className={styles.tileImgContainer}>
+            <img className={styles.tileImg} src={tileImg} alt={""} />;
```

Comment:
Remove stray `;` that is being rendered:

```suggestion
            <img className={styles.tileImg} src={tileImg} alt={""} />
```